src/components/apps/admin/publicationRequests/RequestListing.js

```diff
@@ -0,0 +1,267 @@
+import React, { useState } from "react";
+import { useTranslation } from "i18n";
+import { useQuery, useMutation, queryCache } from "react-query";
+
+import { build, EmptyTable } from "@cyverse-de/ui-lib";
+
+import DETableHead from "components/utils/DETableHead";
+import TableLoading from "components/utils/TableLoading";
+import {
+    getAppPublicationRequests,
+    adminPublishApp,
+    APP_PUBLICATION_REQUESTS_QUERY_KEY,
+} from "serviceFacades/apps";
+import DEDialog from "components/utils/DEDialog";
+import ErrorTypographyWithDialog from "components/utils/error/ErrorTypographyWithDialog";
+
+import ids from "../../ids";
+import constants from "../../constants";
+
+import {
+    Button,
+    IconButton,
+    Paper,
+    TableContainer,
+    Table,
+    TableBody,
+    TableCell,
+    TableRow,
+    Tooltip,
+} from "@material-ui/core";
+import PublicIcon from "@material-ui/icons/Public";
+
+/**
+ *
+ * @author sriram
+ *
+ * A component that displays a list of app publication requests.
+ *
+ */
+
+const appColumnData = (t) => [
+    { name: t("name"), enableSorting: false, key: "name" },
+    {
+        name: t("integratorName").split(":")[0],
+        enableSorting: false,
+        key: "integrator_name",
+    },
+    {
+        name: t("integratorEmail").split(":")[0],
```

Comment:
Does the integrator email and integrator name have `:` in them?

Reply:
Yup. They have `:` in  i18n messages. I didn't want to touch lots of unrelated files or dupe it, so I decided to remove it from the message. I can go ahead and add another entry for these.